src/functions_call.py

```
@@ -0,0 +1,90 @@
+
+
+def get_error_data(methods, file_a, file_b):
+    As, Bs = load_inputs(file_a, file_b)
+    t_rows = {}
+    r_rows = {}
+    for name in methods:
+            try:
+                _, _, t_stats, r_stats = run_method(name, As, Bs)
+                t_rows[name] = t_stats
+                r_rows[name] = r_stats
+            except Exception:
```

Ruff (0.14.8)

Do not catch blind exception: `Exception`

(BLE001)